client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+
+class PluginAutomatedTransfer extends Component {
+
+	static propTypes = {
+		translate: PropTypes.func,
+		plugin: PropTypes.object,
+	}
+
+	static defaultProps = {
+		plugin: {
+			name: 'Plugin',
+		},
+	}
+
+	state = {
+		transferStep: 'start',
+	}
+
+	setNoticeText = ( pluginName = '' ) => {
+		const { translate } = this.props;
+		const { transferStep } = this.state;
+
+		switch ( transferStep ) {
+			case 'start': return translate( 'Installing %s…', {
+				args: pluginName,
+				context: 'Installing plugin…',
```

Comment:
let's be cautious about using `context` in translation strings, only putting it in when we're differentiating between two or more valid meanings. a `comment` here would be useful